.github/workflows/README.md

```diff
@@ -0,0 +1,63 @@
+# Our Workflows
+
+## Format Check
+[![Format Check](https://github.com/nasa/osal/actions/workflows/format-check.yml/badge.svg)](https://github.com/nasa/osal/actions/workflows/format-check.yml)
+
+This action uses clang-format-10 to check for format errors.
+
+## CodeQL Analysis
+[![CodeQL Analaysis](https://github.com/nasa/cfs/actions/workflows/codeql-build.yml/badge.svg)](https://github.com/nasa/cfs/actions/workflows/codeql-build.yml)
+
+This action runs GitHub's industry-leading static analysis engine, CodeQL, against our repository's source code to find security vulnerabilities. It then automatically uploads the results to GitHub so they can be displayed in the repository's code scanning alerts found under the security tab. CodeQL runs an extensible set of [queries](https://github.com/github/codeql), which have been developed by the community and the [GitHub Security Lab](https://securitylab.github.com/) to find common vulnerabilities in your code.
+
+CodeQL is ran for every push and pull-request on all branches of cFS in GitHub Actions.
```

Comment:
Use active voice by replacing "is ran" with "runs". Check rest of documentation for this type of usage

Reply:
Changed to "runs" for all "is ran".